src/sentry/api/serializers/models/release_threshold.py

```diff
@@ -1,9 +1,9 @@
 from sentry.api.serializers import Serializer, register, serialize
+from sentry.models import ReleaseThreshold
```

Comment:
should favor importing directly from the model file. (i.e. import from `sentry.models.releasethreshold`). We should work towards enforcing this somewhere